jordan_bundy/lib/routes.js

```diff
@@ -0,0 +1,49 @@
+var fs = require('fs');
+
+var routes = {};
+var routeList = '';
+
+routes['/'] = {
+  GET: {
+    status: 200,
+    contentType: 'text/html',
+    data: fs.readFileSync(__dirname + '/../public/index.html').toString()
+  }
+};
+
+routes['/name'] = {
+  GET: {
+    status: 200,
+    contentType: 'text/plain',
+    data: 'Hello there Mr. '
+  },
+  POST: {
+    
+  }
+};
+
+routes['/time'] = {
+  GET: {
+    status: 200,
+    contentType: 'text/plain',
+    data: new Date().toString()
+  }
+};
+
+Object.keys(routes).forEach(function(route) {
+  routeList += '<a href="' + route + '">' + route + '</a><br>';
+});
+
+fs.writeFileSync(__dirname + '/../public/index.html', '<!doctype html>'+
+  '<html>'+
+  '<head>'+
+  '</head>'+
+  '<body>'+
+    '<h1>Hello World</h1>'+
+    routeList +
+  '</body>'+
+  '</html>'
+  );
+
+module.exports = routes;
+
```

Comment:
Wow, you were quick! I was just about to push up my post route, but i couldn't get my test to work correctly with it.  i'll push it so it's here, Tyler is saying not to worry about it since we'll be working on post requests more today.